README.md

```diff
@@ -7,916 +7,62 @@
-  - [Layer configuration examples](#layer-configuration-examples)
-  - [Using Pebble](#using-pebble)
-  - [Container usage](#container-usage)
-  - [Layer specification](#layer-specification)
-  - [API and clients](#api-and-clients)
-  - [Hacking / Development](#hacking--development)
-  - [Contributing](#contributing)
+Pebble's key features:
 
-## General model
+- Service management with [layers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)
+- Service [dependencies](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/service-dependencies/)
+- Service [logs](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/logs/) and [log forwarding](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/log-forwarding/)
+- [Health checks](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/health-checks/)
+- [Notices](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/notices/)
+- Identities
+- Can be used in Machines and containers
+- [CLI](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/cli-commands/), [HTTP API and Go client](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/api-and-clients/)
 
-Pebble is organized as a single binary that works as a daemon and also as a
-client to itself. When the daemon runs it loads its own configuration from the
-`$PEBBLE` directory, as defined in the environment, and also records in
-that same directory its state and unix sockets for communication. If that variable
-is not defined, Pebble will attempt to look for its configuration from a default
-system-level setup at `/var/lib/pebble/default`. Using that directory is encouraged
-for whole-system setup such as when using Pebble to control services in a container.
+## Quick start
 
-The `$PEBBLE` directory must contain a `layers/` subdirectory that holds a stack of
-configuration files with names similar to `001-base-layer.yaml`, where the digits define
-the order of the layer and the following label uniquely identifies it. Each
-layer in the stack sits above the former one, and has the chance to improve or
-redefine the service configuration as desired.
+Prerequisites:
 
-## Layer configuration examples
+- A Linux machine.
+- Python 3.x (used to run a basic HTTP server as a sample service managed by Pebble).
 
-Below is an example of the current configuration format.
-For full details of all fields, see the [complete layer specification](#layer-specification).
-
-```yaml
+```bash
+git clone https://github.com/canonical/pebble.git
+cd pebble
+mkdir -p ~/PEBBLE/layers
+export PEBBLE=$HOME/PEBBLE
+echo """\
 summary: Simple layer
-
 description: |
-    A better description for a simple layer.
-
+    A simple layer.
 services:
-    srv1:
+    http-server:
         override: replace
-        summary: Service summary
-        command: cmd arg1 "arg2a arg2b"
+        summary: demo http server
```

Comment:
We could probably leave this line out to help make this more minimal.

Reply:
Yeah, let's remove the top-level summary and description as well.